scripts/sanitycheck

```diff
@@ -1777,9 +1777,17 @@ class CMake():
 
         verbose("Running cmake on %s for %s" %(self.source_dir, self.platform.name))
 
-        cmake_args = ['-B{}'.format(self.build_dir),
-                      '-S{}'.format(self.source_dir),
-                      '-G{}'.format(get_generator()[1])]
+        ldflags="-Wl,--fatal-warnings"
```

Comment:
why is ldflags special?

Reply:
it is not, I copied the code as it was in previous version and modified it slightly while I look into a long term fix and addressing some other options we have to deal with errors and deprecations.